crates/pop-cli/src/commands/call/contract.rs

```diff
@@ -564,6 +605,7 @@ mod tests {
 		// Contract deployed on Pop Network testnet, test get
 		CallContractCommand {
 			path: Some(temp_dir.path().join("testing")),
+			path_pos: Some(temp_dir.path().join("testing")),
```

Comment:
Should not be None? We already specified `path`, no need to specify `path_pos`.  This is probably what is making the unit tests fail, you can test it locally with: `cargo test --lib --bins`.